src/django_otp/plugins/otp_totp/models.py

```diff
@@ -157,16 +158,26 @@ def config_url(self):
         }
         urlencoded_params = urlencode(params)
 
-        issuer = getattr(settings, 'OTP_TOTP_ISSUER', None)
-        if callable(issuer):
-            issuer = issuer(self)
-        if isinstance(issuer, str) and (issuer != ''):
+        issuer = self._read_str_from_settings('OTP_TOTP_ISSUER')
+        if issuer:
             issuer = issuer.replace(':', '')
             label = '{}:{}'.format(issuer, label)
             urlencoded_params += '&issuer={}'.format(
                 quote(issuer)
             )  # encode issuer as per RFC 3986, not quote_plus
 
+        image = self._read_str_from_settings('OTP_TOTP_IMAGE')
+        if image:
+            urlencoded_params += "&image={}".format(quote(image, safe=':/'))
```

Comment:
The inclusion of `:` in the safe characters set is to match the tested logic I'm currently using to append this parameter to the URL string before making a QR code.